cmd/kubectl-package/command/build.go

```diff
@@ -75,7 +75,11 @@ func (b Build) Run(ctx context.Context) error {
 
 	verboseLog.Info("validating package image")
 
-	structureLoaderOpts := []packagestructure.LoaderOption{packagestructure.WithManifestValidators(&packagestructure.ObjectPhaseAnnotationValidator{})}
+	structureLoaderOpts := []packagestructure.LoaderOption{
+		packagestructure.WithManifestValidators(
+			packagestructure.DefaultValidators,
```

Comment:
Since there are now three users of this piece of code (build subcmd, validate subcmd and mage pkg builder in my PR) ... maybe we could have a "gimme leader with default validator" shortcut for this?